### PR TITLE
Update from template repository + minor Angular fixes

### PR DESCRIPTION
**Pull request by williamvinogradov**
Merged the last updates from the https://github.com/DevExpress-Examples/DevExtreme-Examples-Template repo.

Also, some minor changes in the Angular.

### CODE REVIEW

File: Angular/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@angular/compiler-cli": "^15.1.0",
     "@types/jasmine": "~4.3.0",
     "eslint": "^8.39.0",
-    "eslint-config-devextreme": "1.1.2",
+    "eslint-config-devextreme": "^1.1.3",
```
Comment (pomahtri) [inline Angular/package.json]:
```suggestion
    "eslint-config-devextreme": "1.1.3",
```

Comment (williamvinogradov) [reply Angular/package.json]: Eup, agree with you.
I missed it :)

Comment (pomahtri) [reply Angular/package.json]: marking as unresolved, since other frameworks also have "^"

Comment (williamvinogradov) [reply Angular/package.json]: Yup, I didn't want to resolve it with GitHub :)
I fixed it in the examples-template repo and merge these changes to this PR

**Review by pomahtri — APPROVED** [review]

**Pull request merged**